packages/react-core/src/components/Menu/MenuGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Menu/menu';
+import { css } from '@patternfly/react-styles';
+
+export interface MenuGroupProps extends React.HTMLProps<HTMLElement> {
+  /** Items within group */
+  children?: React.ReactNode;
+  /** Additional classes added to the MenuGroup */
+  className?: string;
+  /** Group label */
+  label?: string;
+  /** ID for title label */
+  titleId?: string;
+  /** Forwarded ref */
+  innerRef?: React.Ref<any>;
+}
+
+const MenuGroupBase: React.FunctionComponent<MenuGroupProps> = ({
+  children = [] as React.ReactElement[],
+  className = '',
+  label = '',
+  titleId = '',
+  innerRef,
+  ...props
+}: MenuGroupProps) => (
+  <section {...props} className={css('pf-c-menu__group', className)} ref={innerRef}>
```

Comment:
is this class in the styles obj?

Reply:
no, just `menuGroupItem`, but no `menuGroup`